_committees/07_08.md

```diff
@@ -24,6 +24,10 @@ committee:
     name: Maia Gibbs
   - role: External Events Coordinator
     name: Sam Turnbull
+    note: Stepped down during the year
+  - role: External Events Coordinator
+    name: Simon Thompson
+    note: Took over after Sam Turnbull stepped down
```

Comment:
Can we add a `(dates unknown)` to these as a mini call to action for more info? Thoughts?